package.json

```diff
@@ -130,6 +130,7 @@
     "oauth-1.0a": "2.2.6",
     "otplib": "12.0.1",
     "pac-proxy-agent": "7.0.1",
+    "pinyin-pro": "^3.19.5",
```

Comment:
The dependency is required by the [docs only](https://github.com/DIYgod/RSSHub/blob/cdc36d3554789bce4d7e457906f4871f08c11a54/website/package.json#L28). It should not be added to RSSHub's `package.json`